AspNetCoreApplication/Extensions/StringExts.cs

```diff
@@ -0,0 +1,20 @@
+﻿using AspNetCoreApplication.Config;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace AspNetCoreApplication.Extensions
+{
+    public static class StringExts
+    {
+        public static string Encrypt(this string password)
+        {
+            var tokenConfig = new TokenConfig();
+            password += tokenConfig.Key;
+            var passwordBytes = Encoding.UTF8.GetBytes(password);
+            return Convert.ToBase64String(passwordBytes);
```

Comment:
Cái này chỉ đơn thuần là mã hóa Base64, nhìn thì đúng là khác mật khẩu gốc nhưng lại không an toàn, có thể bị dịch ngược

Tham khảo cái này 

https://forums.asp.net/t/1847275.aspx?Hashing+a+string+using+MD5+and+with+Salt

Reply:
done